Cargo.lock

```diff
@@ -2428,6 +2426,15 @@ dependencies = [
  "either",
 ]
 
+[[package]]
+name = "itertools"
+version = "0.11.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "b1c173a5686ce8bfa551b3563d0c2170bf24ca44da99c7ca4bfdab5418c3fe57"
+dependencies = [
+ "either",
+]
```

Comment:
unfortunate that we get yet another instance of itertools. Not a biggy I suppose

Reply:
Yeah, I figured it would be fine since it is mostly monomorphized code anyhow

Reply:
I've update itertools to 0.11 everywhere now. We only have 0.10 for the objectron examples, via to the `prost` crate